src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/Referral.kt

```diff
@@ -24,6 +26,10 @@ data class Referral(
   var whenUnavailable: String? = null,
   var complexityLevelID: UUID? = null,
   var serviceCategoryID: UUID? = null,
+
+  @OneToMany @JoinColumn(name = "service_category_id")
```

Comment:
i don't think this is required since the desired outccomes are set on the referral object explicitly via a call to PATCH /draft-referral

Reply:
Updated